unstructured/partition/utils/config.py

```diff
@@ -117,5 +140,28 @@ def PDF_ANNOTATION_THRESHOLD(self) -> float:
 
         return self._get_float("PDF_ANNOTATION_THRESHOLD", 0.9)
 
+    @property
+    def STORAGE_ENABLED(self) -> bool:
+        """Enable usage of STORAGE_DIR and STORAGE_TMPDIR."""
+        return self._get_bool("STORAGE_ENABLED", False)
+
+    @property
+    def STORAGE_DIR(self) -> str:
+        """Path to Unstructured storage directory."""
```

Comment:
STORAGE_DIR is a misleading name, which has permanent or at least caching connotations. could this instead be TMP_STORAGE_DIR?

Reply:
There are parameters `STORAGE_DIR` and `STORAGE_TMPDIR`

Reply:
Maybe `UNSTRUCTURED_DIR` and `UNSTRUCTURED_TMPDIR` as those point to `~/.cache/unstructured` and `~/.cache/unstructured/tmp/{gid}` respectively

Reply:
or `UNSTRUCTURED_CACHE_DIR` and `UNSTRUCTURED_TMP_DIR`?